src/Actions/DeleteMultisiteEntry.php

```diff
@@ -18,6 +20,10 @@ public function visibleTo($item)
 
     public function fieldItems()
     {
+        if (! $this->canChangeBehaviour()) {
+            return [];
+        }
+
```

Comment:
When you try to delete an entry where you don't have permission to access each site, you no longer get the option whether to delete or detach. Great!

But, when you try to delete an entry that has no localizations at all, it _does_ ask you.

I know that prior to this PR we would ask you regardless anyway, but now that sometimes you get the option and sometimes you don't, it feels odd.

We should avoid showing the option when there are no localizations.

Reply:
Good catch!